src/input/mpegts/iptv/iptv_rtsp.c

```diff
@@ -224,7 +262,7 @@ iptv_rtsp_read ( iptv_mux_t *im )
     r = recv(im->mm_iptv_fd2, buf, sizeof(buf), MSG_DONTWAIT);
   } while (r > 0);
 
-  r = iptv_rtp_read(im, um, NULL);
+  r = iptv_rtp_read(im, um, & iptv_rtp_header_callback);
```

Comment:
I think that & is not required here. Functions are always converted as pointers.